libraries/AP_Motors/AP_MotorsHeli.cpp

```diff
@@ -117,6 +110,42 @@ const AP_Param::GroupInfo AP_MotorsHeli::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("OPTIONS", 28, AP_MotorsHeli, _heli_options, (uint8_t)HeliOption::USE_LEAKY_I),
 
+    // @Param: COL_MIN_DEG
+    // @DisplayName: Minimum Collective Blade Pitch Angle
```

Comment:
For the display name I would try and move "Minimum" to the end.

Reply:
Is there a standard that I can look at to follow.  I would assume that you would want me to do the same with maximum collective blade pitch angle and others potentially.

Reply:
This is the best we have but sadly the example it gives has "Maximum" at the beginning. https://ardupilot.org/dev/docs/style-guide.html#display-name

Reply:
It's not a blocker though of course..

Reply:
Just offering another idea: we don't actually need the deg as the units are specified in the meta data.  So COL_ANG_MIN may be a better name?

Reply:
fixed.  thanks!